tests/CMakeLists.txt

```diff
@@ -130,11 +131,6 @@ if(NOT USE_CRYPTO_SUBMODULE)
     add_test_suite(rsa)
     add_test_suite(xtea)
 endif()
-add_test_suite(debug)
```

Comment:
Why were these tests removed? IS it a rebase issue?

Reply:
Rebase error, sorry about that. Fixed now.